interface/reports/receipts_by_method_report.php

```diff
@@ -588,7 +589,7 @@ function sel_procedure() {
                 if (empty($row['session_id'])) {
                     $rowmethod = trim($row['memo']);
                 } else {
-                    $rowmethod = trim($row['payment_method']);
+                    $rowmethod = trim($row['title']);
```

Comment:
Rather than querying list_options, much better to use the following function on the option_id (will collect the title from an option_id and also translate it):
https://github.com/openemr/openemr/blob/master/library/options.inc.php#L3756